src/status_im/signals/core.cljs

```diff
@@ -54,9 +56,12 @@
                 :peers-count (count (:peers peer-stats)))}))
 
 (defn handle-local-pairing-signals
-  [signal-type]
+  [{:keys [type action] :as event}]
   (log/info "local pairing signal received"
```

Comment:
now that we know what type of signal is received, is this log/info absolutely necessary?

Reply:
emm.. maybe we can use log/debug instead log/info? @siddarthkay

Reply:
`log/debug` makes sense if you plan to use this log in the future.

else I would say you can get rid of it and only add it during your local development process.
I don't have a strict preference, I leave it up to your judgement :)